.gitattributes

```diff
@@ -1 +1,2 @@
 CHANGES.md merge=union
+*.ps1 text=auto eol=lf
```

Comment:
Why do we need this, Sergii?

Reply:
`build_genconf_windows.ps1` contains Linux shell template which is then filled up with a values. The idea is to replace `CRLF` chars with a `LF` in order to run the script on Linux shell.

Reply:
Could this be done as part of a filtering step after the template replacement, to allow PowerShell scripts to keep their expected newlines? This would prevent problems if someone manually edits the script on a Windows node - first they'll see the code correctly, and second, if they introduce Windows newlines, things will continue to work.

Reply:
removed this part, and doing ```(Get-Content -Path .\gen\build_deploy\powershell\dcos_generate_config_win.sh.in -Raw) -Replace "`r`n", "`n" | Set-Content -NoNewline -Path "$($artifact_out)";```